manager/MaCh3Modes.h

```diff
@@ -0,0 +1,79 @@
+#pragma once
+
+#include <cmath>
+#include <map>
+#include <string>
+
+//MaCh3 includes
+#include "manager/YamlHelper.h"
+#include "manager/Monitor.h"
+#include "manager/MaCh3Exception.h"
+
+/// Enumerator of MaCh3Mode
+using MaCh3Modes_t = int;
+
+/// @brief KS: Class containing information for a single MaCh3Mode
```

Comment:
Very minor, but shall we call it ModeInfo or just Mode? Or MaCh3Mode? There are many of these which then make up one "MaCh3Modes" object so I am wondering if just "Mode" is clearer?